vimeo-networking/src/main/java/com/vimeo/networking/callbacks/IgnoreResponseVimeoCallback.java

```diff
@@ -22,24 +24,18 @@
 
 package com.vimeo.networking.callbacks;
 
-import org.jetbrains.annotations.NotNull;
-
 /**
- * Abstract callback class for {@code T} type of model.
+ * Use this callback when you don't care about the server response and therefore don't need to deal
+ * with typed network calls.
  * <p>
- * Created by hanssena on 4/27/15.
+ * Created by rigbergh on 7/20/17.
  */
-public abstract class ModelCallback<T> extends VimeoCallback<T> {
+public abstract class IgnoreResponseVimeoCallback extends VimeoCallback<Object> {
```

Comment:
Why should a consumer use this class over `VimeoCallback`? It seems like an unnecessary duplication which introduces complexity when consuming any api since the existence of the object in the success method shouldn't cause problems if the consumer doesn't use it.